packages/lavapack/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@lavamoat/lavapack",
-  "version": "3.2.1",
+  "version": "3.3.0",
```

```suggestion
  "version": "4.0.0",
```

As cee9caf3fd39bb943c9b0437f5bfe1a0982bdaa8 constitutes a breaking change of this package, this should be a semver-major.